index.html

```diff
@@ -33,7 +33,8 @@
 
 <body>
     <label for="cipher-input">entry voor het Nerdland gastenboek</label><br>
-    <textarea id="cipher-input" maxlength="120" cols="31" rows="4"></textarea><br>
+    <textarea id="user-name" placeholder="naam" cols="31" rows="1"></textarea><br>
```

Comment:
`<input type="text" id="user-name" placeholder="naam" />` lijkt mij correcter (en gemakkelijker)
als we daarvoor ook nog eens `<label for="user-name">Discord username</label>` zetten, is het ook accessible

Reply:
fixed